### PR TITLE
Add remove button to todo items

### PR DESCRIPTION
**Pull request by cetinsuheyla**

### CODE REVIEW

File: src/InputWithLabel.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React, {useRef, useEffect} from 'react'
+
+function InputWithLabel({todoTitle, handleTitleChange, children}) {
```
Comment (davidrapoport) [inline src/InputWithLabel.js:3]: You don't need to add `children` as a prop. It's a default prop that React understands.

File: src/App.js [hunk 2]
```diff
@@ -20,13 +20,16 @@ function App() {
 		setTodoList([...todoList, newTodo]);
 	}
 
+  function removeTodo(id){
```
Comment (davidrapoport) [inline src/App.js:23]: nit: indentation

Comment (davidrapoport) [reply src/App.js:23]: also, when is this function used?

File: src/InputWithLabel.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  const inputRef = useRef();
+  useEffect(()=> {inputRef.current.focus()})
+  return (
+    <div>
+      <label htmlFor="todoTitle">{children}</label>
+				<input
+          ref={inputRef}
```
Comment (davidrapoport) [inline src/InputWithLabel.js:10]: hey not sure if it's just my new computer being weird here or if the indentation here is off as well.

File: src/App.js [hunk 4]
```diff
@@ -20,13 +20,16 @@ function App() {
...
+    setTodoList(filteredTodos);
+  }
+
 	return (
 		<>
 			<h1>Todo List</h1>
 			<AddTodoForm onAddTodo={addTodo} />
 
-			<TodoList todoList={todoList} />
-		</>
```
Comment (davidrapoport) [inline src/App.js:29]: I don't think you want to remove the fragment close tag here.

**Review by davidrapoport — CHANGES_REQUESTED** [review]
Hey Suheyla, you're on the right track but there are a few things that need fixing here. It seems like maybe you forgot to add a commit or something? Let me know if you need any help!

**Comment by cetinsuheyla** [comment]
Hi David,

Yes, probably I missed a commit here. I will check it out and publish new
changes. Oh, the indentation problem is because I needed to
uninstall prettier because of my work requirements, but I will install it
for assignments from now on.

Thanks,
Suheyla

**Comment by cetinsuheyla** [comment]
Hi David,

I tried to check every instruction and applied them into code.
Here is the new pull request:
https://github.com/cetinsuheyla/ctd-react-canary/pull/5
I also put a little note to the pull request.
Please let me know if I need to change anything.

Thanks,
Suheyla

**Pull request closed without merging**